data_validation/validation_builder.py

```diff
@@ -265,6 +329,7 @@ def get_source_query(self):
         }
         query = self.source_builder.compile(**source_config)
         if self.verbose:
+            print(source_config)
```

Comment:
Let's make sure we print the target_config in the `get_target_query(self)` method too for standardization